### PR TITLE
Deprecating wallet picker

### PR DESCRIPTION
**Pull request by bangtoven**

### CODE REVIEW

File: packages/wallet-sdk/src/components/ConnectContent/ConnectContent.tsx [hunk 1]
```diff
@@ -8,12 +8,10 @@ import { useCallback, useState } from 'preact/hooks';
 import { createQrUrl } from '../../util';
```
Comment (bangtoven) [inline packages/wallet-sdk/src/components/ConnectContent/ConnectContent.tsx:8]: before
Screenshot 2023-10-27 at 4 59 15 PM

after
image

File: packages/wallet-sdk/src/components/icons/QRCodeIcon.tsx [hunk 2]
```diff
@@ -2,16 +2,11 @@ import { h } from 'preact';
...
-      <path d="M8.2271 1.77124L7.0271 1.77124V2.97124H8.2271V1.77124Z" />
-      <path d="M5.44922 0.199219L5.44922 4.54922L9.79922 4.54922V0.199219L5.44922 0.199219ZM8.89922 3.64922L6.34922 3.64922L6.34922 1.09922L8.89922 1.09922V3.64922Z" />
-      <path d="M2.97124 1.77124L1.77124 1.77124L1.77124 2.97124H2.97124V1.77124Z" />
-      <path d="M0.199219 4.54922L4.54922 4.54922L4.54922 0.199219L0.199219 0.199219L0.199219 4.54922ZM1.09922 1.09922L3.64922 1.09922L3.64922 3.64922L1.09922 3.64922L1.09922 1.09922Z" />
-      <path d="M2.97124 7.0271H1.77124L1.77124 8.2271H2.97124V7.0271Z" />
-      <path d="M0.199219 9.79922H4.54922L4.54922 5.44922L0.199219 5.44922L0.199219 9.79922ZM1.09922 6.34922L3.64922 6.34922L3.64922 8.89922H1.09922L1.09922 6.34922Z" />
-      <path d="M8.89922 7.39912H7.99922V5.40112H5.44922L5.44922 9.79912H6.34922L6.34922 6.30112H7.09922V8.29912H9.79922V5.40112H8.89922V7.39912Z" />
-      <path d="M7.99912 8.89917H7.09912V9.79917H7.99912V8.89917Z" />
-      <path d="M9.79917 8.89917H8.89917V9.79917H9.79917V8.89917Z" />
+    <svg width="18" height="18" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" {...props}>
```
Comment (bangtoven) [inline packages/wallet-sdk/src/components/icons/QRCodeIcon.tsx:5]: before
image

after
image

File: packages/wallet-sdk/src/relay/MobileRelay.ts [hunk 3]
```diff
@@ -25,9 +25,7 @@ export class MobileRelay extends WalletLinkRelay {
     return {
       promise: new Promise(() => {
         const location = getLocation();
-        location.href = `https://www.coinbase.com/connect-dapp?uri=${encodeURIComponent(
-          location.href
-        )}`;
+        location.href = `https://go.cb-w.com/dapp?cb_url=${encodeURIComponent(location.href)}`;
```
Comment (bangtoven) [inline packages/wallet-sdk/src/relay/MobileRelay.ts:28]: reverting 
image

`dapp` is also a correct path

File: packages/wallet-sdk/src/components/ConnectContent/ConnectContent.tsx (hunk 1 above)
Comment (bangtoven) [reply packages/wallet-sdk/src/components/ConnectContent/ConnectContent.tsx:8]: fyi, we are working with design for new UI, removing the white space and making it responsive to screensize.

File: packages/wallet-sdk/src/relay/MobileRelay.ts (hunk 3 above)
Comment (cb-jake) [reply packages/wallet-sdk/src/relay/MobileRelay.ts:28]: Do we need any walletlink changes?

**Review by cb-jake — COMMENTED** [review]
if we're only supporting a single wallet moving forward we can clean up the render on ConnectContent by removing the mapping of the defined wallets and just rending the CoinbaseWalletSteps directly with a single ConnectItem component.

**Comment by bangtoven** [comment]
thanks @cb-jake for the review!
- no walletlink changes needed for this to work.
- will take a look at `setAppSrc`
- gotcha. let me see if there's any more clean up we could do. fyi, actually we asked design/product support on this to make the UI look better with single wallet.

File: packages/wallet-sdk/src/components/Snackbar/Snackbar.tsx [hunk 4]
```diff
@@ -7,18 +7,9 @@ import { useEffect, useState } from 'preact/hooks';
...
...QxOWE4LjkzIDguOTMgMCAwMTguOTMtOC45M2gxMS4xNjNhOC45MyA4LjkzIDAgMDE4LjkzIDguOTN2MTEuMTYzYTguOTMgOC45MyAwIDAxLTguOTMgOC45M0gxMC40MjJhOC45MyA4LjkzIDAgMDEtOC45My04LjkzVjEwLjQxOXoiIGZpbGw9IiMxNjUyRjAiLz48cGF0aCBmaWxsLXJ1bGU9ImV2ZW5vZGQiIGNsaXAtcnVsZT0iZXZlbm9kZCIgZD0iTTEwLjQxOSAwSDIxLjU4QzI3LjMzNSAwIDMyIDQuNjY1IDMyIDEwLjQxOVYyMS41OEMzMiAyNy4zMzUgMjcuMzM1IDMyIDIxLjU4MSAzMkgxMC40MkM0LjY2NSAzMiAwIDI3LjMzNSAwIDIxLjU4MVYxMC40MkMwIDQuNjY1IDQuNjY1IDAgMTAuNDE5IDB6bTAgMS40ODhhOC45MyA4LjkzIDAgMDAtOC45MyA4LjkzdjExLjE2M2E4LjkzIDguOTMgMCAwMDguOTMgOC45M0gyMS41OGE4LjkzIDguOTMgMCAwMDguOTMtOC45M1YxMC40MmE4LjkzIDguOTMgMCAwMC04LjkzLTguOTNIMTAuNDJ6IiBmaWxsPSIjZmZmIi8+PHBhdGggZmlsbC1ydWxlPSJldmVub2RkIiBjbGlwLXJ1bGU9ImV2ZW5vZGQiIGQ9Ik0xNS45OTggMjYuMDQ5Yy01LjU0OSAwLTEwLjA0Ny00LjQ5OC0xMC4wNDctMTAuMDQ3IDAtNS41NDggNC40OTgtMTAuMDQ2IDEwLjA0Ny0xMC4wNDYgNS41NDggMCAxMC4wNDYgNC40OTggMTAuMDQ2IDEwLjA0NiAwIDUuNTQ5LTQuNDk4IDEwLjA0Ny0xMC4wNDYgMTAuMDQ3eiIgZmlsbD0iI2ZmZiIvPjxwYXRoIGQ9Ik0xMi43NjIgMTQuMjU0YzAtLjgyMi42NjctMS40ODkgMS40ODktMS40ODloMy40OTdjLjgyMiAwIDEuNDg4LjY2NiAxLjQ4OCAxLjQ4OXYzLjQ5N2MwIC44MjItLjY2NiAxLjQ4OC0xLjQ4OCAxLjQ4OGgtMy40OTdhMS40ODggMS40ODggMCAwMS0xLjQ4OS0xLjQ4OHYtMy40OTh6IiBmaWxsPSIjMTY1MkYwIi8+PC9zdmc+`;
```
Comment (bangtoven) [inline packages/wallet-sdk/src/components/Snackbar/Snackbar.tsx:10]: reverting previous change https://github.com/coinbase/coinbase-wallet-sdk/commit/b0c9041a1fe2ff6d37291395696fa3cbeb2ad518#diff-5963d849ba8821a176ba59be3d5c0ce674b78312019d36635730546fe81b296a

File: packages/wallet-sdk/src/relay/MobileRelay.ts (hunk 3 above)
Comment (bangtoven) [reply packages/wallet-sdk/src/relay/MobileRelay.ts:28]: no. this is handled by branch.io

**Review by cb-jake — APPROVED** [review]

**Pull request merged**